sdks/python/pom.xml

```diff
@@ -56,6 +56,22 @@
     </pluginManagement>
     <plugins>
       <plugin>
+        <artifactId>maven-clean-plugin</artifactId>
+        <version>3.0.0</version>
```

Comment:
can all this be moved into the root `pom.xml` `pluginManagement` section?

Specifically, the `version` should be inherited, but I don't think any of this necessarily applies only to the `sdks`  module. (E.g., we might have examples, runners, etc. someday.)